Libraries/src/net/relinc/libraries/data/Dataset.java

```diff
@@ -1,12 +1,29 @@
 package net.relinc.libraries.data;
 
 public class Dataset {
-	public double[] data;
-	public double[] timeData;
-	
+	private double[] data;
+	private double[] timeData;
```

Comment:
an overall good measure to take. The `get` functions return a `clone` of the array, so these can never change. Before they could've gotten changed by anyone

Reply:
I've never looked at multiple viewers, if it's a common workflow we should look into it